src/Wallabag/CoreBundle/EventListener/AuthenticationListener.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Wallabag\CoreBundle\EventListener;
+
+use FOS\UserBundle\FOSUserEvents;
+use Symfony\Component\DependencyInjection\Container;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use FOS\UserBundle\Event\FilterUserResponseEvent;
+use Wallabag\CoreBundle\Entity\Config;
+
+class AuthenticationListener implements EventSubscriberInterface
+{
+    private $em;
+    private $container;
+
+    public function __construct(Container $container, $em)
+    {
+        $this->container = $container;
+        $this->em = $em;
+    }
+
+    public static function getSubscribedEvents()
+    {
+        return array(
+            FOSUserEvents::REGISTRATION_CONFIRMED => 'authenticate',
+        );
+    }
+
+    public function authenticate(FilterUserResponseEvent $event, $eventName = null, EventDispatcherInterface $eventDispatcher = null)
+    {
+        if (!$event->getUser()->isEnabled()) {
+            return;
+        }
+
+        $config = new Config($event->getUser());
```

Comment:
Could you put a comment on that class to explain that it is called after a user is registered?
Or maybe renamed it to `RegistratioConfirmedListener` to be more explicit.